bjoern/server.c

```diff
@@ -224,6 +249,12 @@ ev_io_on_read(struct ev_loop* mainloop, ev_io* watcher, const int events)
         request->current_chunk = _PEP3333_Bytes_FromString(
           http_error_messages[HTTP_SERVER_ERROR]);
       }
+    } else if (request->state.expect_continue) {
+      read_state = expect_continue;
+    } else if (request->state.expectation_failed) {
+      read_state = done;
+      request->current_chunk = _PEP3333_Bytes_FromString(
+        EXPECTATION_FAILED);
```

Comment:
We already have `http_error_messages[...]`, maybe you can move `EXPECTATION_FAILED` there.